src/components/InputLabel/InputLabel.tsx

```diff
@@ -45,12 +46,16 @@ const InputLabel: FC<InputLabelProps> = ({
   sx,
   noMinWidth,
   htmlFor,
+  helpTip,
+  helpTipPlacement,
   ...props
 }) => {
   return (
     <CustomLabel sx={sx} htmlFor={htmlFor} {...props}>
       <span className={`${noMinWidth ? "noMinWidthLabel" : ""}`}>
-        {children}
+        <HelpTip placement={helpTipPlacement} content={helpTip}>
+          {children}
+        </HelpTip>
```

Comment:
We need to add a validation here to not throw the component in case no help tip is set

```suggestion
        {helpTip ? <HelpTip placement={helpTipPlacement} content={helpTip}>
          {children}
        </HelpTip> : children}
```